src/components/GamesList/GamesList.jsx

```diff
@@ -0,0 +1,169 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getGameByCategory, getGames } from '../../redux/gamesSlice/gamesSlice';
+
+import shooterImg from '../../assets/img/shooter/shooter.jpg';
+import mmorpgImg from '../../assets/img/mmorpg/mmorpg.jpg';
+import strategyImg from '../../assets/img/strategy/strategy.jpg';
+import fightingImg from '../../assets/img/fighting/fighting.jpg';
+import battleRoyaleImg from '../../assets/img/battleRoyale/battleRoyale.jpg';
+import mobaImg from '../../assets/img/moba/moba.jpg';
+import sportsImg from '../../assets/img/sports/sports.jpg';
+import cardGameImg from '../../assets/img/cardGame/cardGame.jpg';
+import racingImg from '../../assets/img/racing/racing.jpg';
+import socialImg from '../../assets/img/social/social.jpg';
+
+const genreBackgroundConfig = {
+  Shooter: shooterImg,
+  Mmorpg: mmorpgImg,
+  Strategy: strategyImg,
+  Fighting: fightingImg,
+  'Battle-Royale': battleRoyaleImg,
+  Moba: mobaImg,
+  Sports: sportsImg,
+  'Card Game': cardGameImg,
+  Racing: racingImg,
+  Social: socialImg,
+};
+
+const gamesGenres = ['Shooter', 'Mmorpg', 'Strategy', 'Fighting', 'Moba', 'Sports', 'Racing', 'Social'];
+
+const GamesList = () => {
+  const games = useSelector((state) => state.games.games);
+  const dispatch = useDispatch();
+
+  const filterCategoryLength = (genre) => (
+    games.filter((game) => game.genre.toLowerCase() === genre.toLowerCase()).length
+  );
+
+  useEffect(() => {
+    dispatch(getGames());
+  }, [dispatch]);
+
+  const handleDetailsPage = (genre) => () => {
+    dispatch(getGameByCategory(genre));
+  };
+
+  return (
+    <Section>
+      <H2>
+        Category
+      </H2>
+      <GridContainer>
+        {gamesGenres.map((game) => (
+          <Card key={game}>
+            <Button
+              type="button"
+              style={{
+                backgroundImage: `url(${genreBackgroundConfig[game]})`,
+              }}
+            >
+              <StyledLink to={`/Details/${game}`}>
+                <GameOverlay onClick={handleDetailsPage(game)}>
+                  <Category>
+                    {game}
+                    {` (${filterCategoryLength(game)})`}
+                  </Category>
+                </GameOverlay>
+              </StyledLink>
+            </Button>
+          </Card>
+        ))}
+      </GridContainer>
+    </Section>
+  );
+};
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: #e0decf;
+`;
+const Section = styled.section`
```

Comment:
- [ ] Great job with the design. At the moment, you have missed some aspects of the design. As per given design, you have not respected the layout (space between elements). You have a gab between the gamelist elements and the borders are rounded which is not the case with the give design.
**Given design**
![Screen Shot 2023-09-06 at 5 37 47 AM](https://github.com/JavierAybar/React_Games_Gallery/assets/62849872/efea8b4a-0b8f-4135-a1ec-1ea0d962d0f4)
**Your design**
![Screen Shot 2023-09-06 at 5 39 37 AM](https://github.com/JavierAybar/React_Games_Gallery/assets/62849872/0f9dd49e-f467-4a29-ba33-ea27a89cbaf3)